### DISCUSSION TITLE
pick list filter does not reset the filter value on dialog box close

### DISCUSSION THREAD

**Discussion by Dileep2506** [primefaces/community#1235] [PrimeNG]
<p-pickList
      #pl
      [source]="list"
      sourceFilterPlaceholder="{{ selectedOptions.selectedValueInfo }}"
      targetFilterPlaceholder="{{ selectedOptions.selectedValueInfo }}"
      filterBy="label"
      [target]="selectedOptions.selectedList"
      sourceHeader="Available {{ selectedOptions.selectedValueInfo }}"
      targetHeader="Selected {{ selectedOptions.selectedValueInfo }}"
      [responsive]="true"
      [sourceStyle]="{ height: '190px' }"
      [targetStyle]="{ height: '190px' }"
      (onMoveToTarget)="onMoveToTarget()"
      (onMoveToSource)="onMoveToSource()"
      [showSourceControls]="false"
      [showTargetControls]="false">
      <ng-template let-masterDataList pTemplate="item">
        <div class="ui-helper-clearfix" *ngIf="!showGroupersView">
          <div
            style="font-size: 14px; float: left; margin: 15px 5px 0 0"
            title="{{ masterDataList.description }}">
            {{ masterDataList.label }}
          </div>
        </div>
        <div class="row" *ngIf="showGroupersView">
          <div class="col-11">
            <div
              style="font-size: 14px; float: left; margin: 15px 5px 0 0"
              title="{{ masterDataList.description }}">
              {{ masterDataList.label }}
            </div>
          </div>
          <div class="col-1">
            <div style="font-size: 14px; float: left; margin: 15px 5px 0 0">
              <button
                pButton
                type="button"
                icon="pi pi-eye"
                class="roundButton ui"
                title="View Codes"
                style="float: right; margin-top: -1%"
                (click)="
                  onViewGroupers(masterDataList, selectedOptions)
                "></button>
            </div>
          </div>
        </div>
      </ng-template>
    </p-pickList>

**Comment by Dileep2506** [comment]
unable to reset the filter value in pick list

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
